typed-builder-macro/src/struct_info.rs

```diff
@@ -188,9 +188,9 @@ impl<'a> StructInfo<'a> {
             #[must_use]
             #builder_type_doc
             #[allow(dead_code, non_camel_case_types, non_snake_case)]
-            #builder_type_visibility struct #builder_name #b_generics {
+            #builder_type_visibility struct #builder_name #b_generics #b_generics_where_extras_predicates {
```

Comment:
BTW - why is this needed? `b_generics_where_extras_predicates` should already be contained inside `b_generics`.

Reply:
I had to add this because it didn't add the `where T: ?Sized` clause.  
Looking at the [`ToTokens` impl](https://github.com/dtolnay/syn/blob/e20379473b266ec5904bc87415d80de3ab1ac059/src/generics.rs#L980-L1013) of `Generics`, the `where` clause is absent, so it has to be added manually.

Reply:
This is certainly weird. I'll look into it locally after merging.